templates/search/search_aggregates_v8/query.sql

```diff
@@ -8,6 +9,9 @@ SELECT
   search_cohort,
   source,
   default_search_engine,
+  os,
+  os_version,
+  COUNT(DISTINCT client_id) as client_count,
```

Comment:
reading from `search_clients_daily_v8` means we already every row has a `DISTINCT client_id`, so we should use `COUNT(*)` here instead:
```suggestion
  COUNT(*) as client_count,
```